packages/trader/src/Stores/Modules/Trading/Actions/start-date.ts

```diff
@@ -1,32 +1,25 @@
 import { ContractType } from 'Stores/Modules/Trading/Helpers/contract-type';
 import { TTradeStore } from 'Types';
 
-type TOnChangeStartDate = (
-    store: TTradeStore
-) => Promise<
-    Pick<
-        TTradeStore,
-        | 'contract_start_type'
-        | 'duration_units_list'
-        | 'duration_min_max'
-        | 'duration_unit'
-        | 'start_time'
-        | 'expiry_date'
-        | 'expiry_type'
-    > & { sessions?: TTradeStore['sessions'] }
+type TOnChangeStartDate = (store: TTradeStore) => Promise<
+    Pick<TTradeStore, 'contract_start_type' | 'duration_units_list' | 'start_time' | 'expiry_date' | 'expiry_type'> & {
+        sessions?: TTradeStore['sessions'];
+    } & {
+        duration_min_max: { [key: string]: { min: number; max: number } } | { min: number; max: number };
```

Comment:
@kate-deriv please update duration_min_max & duration_unit types in TOverrideTradeStore instead

Reply:
I think I'll remove this type because TS is determining the returned value correctly (probably because ContractType  packages/trader/src/Stores/Modules/Trading/Helpers/contract-type.ts is in TS now)